datawig/imputer.py

```diff
@@ -760,8 +761,13 @@ def predict(self,
                                     imputation
         :param imputation_suffix: suffix for imputation columns
         :param score_suffix: suffix for imputation score columns
+        :param inplace: whether to add columns to passed DataFrame.
```

Comment:
Maybe we can be a bit more specific here? As in, we will add an `imputed` column and `imputed_proba`.

Reply:
good call. done in the new pull request: https://github.com/awslabs/datawig/pull/55
(not mentioning the names explicitly because they can independently be modified.)